ApplicationLibCode/ProjectDataModel/Summary/RimEnsembleCurveSet.h

```diff
@@ -95,8 +95,11 @@ class RimEnsembleCurveSet : public caf::PdmObject, public RimEnsembleCurveSetInt
 
     QString name() const;
 
-    bool isCurvesVisible();
-    void setColor( cvf::Color3f color );
+    bool   isCurvesVisible();
```

Comment:
isCurvesVisible sounds like it should be const?

Reply:
Fixed